README.md

```diff
@@ -16,17 +16,63 @@
+
+In all IT systems, no matter the domain, decisions are made all the time. Which offer should a customer get? Who should receive a marketing message? Does this device need servicing? Is this a fraud?
+
+Algorithms for making such decisions can be developed by programmers. With Nussknacker however, such decision algorithms can be authored and deployed without the need to involve IT.
+
+An essential part of Nussknacker is a visual design tool for decision algorithms. It allows not-so-technical users, like analysts or business people, to author decision logic in an imperative, easy-to-follow and understandable way. Once authored, with a click of a button, scenarios are deployed for execution. And can be changed and redeployed anytime there’s a need.
+
+Nussknacker supports three [processing modes](https://nussknacker.io/documentation/about/ProcessingModes/): streaming, request-response and batch (planned in version 1.15). In streaming mode, Nussknacker uses Kafka as its primary interface: input streams of data and output streams of decisions. In request-response mode, it exposes HTTP endpoints with OpenAPI definitions. And since input events or requests are often not enough to make a decision, Nussknacker can enrich them with data from databases and OpenAPI endpoints.
+
+Nussknacker can be used with its own, lightweight, Kubernetes-based [engine](https://nussknacker.io/documentation/about/engines/). It provides scalability and high availability while being straightforward to set up and maintain. However, in advanced streaming use cases, when aggregations in time windows are needed, Nussknacker can execute decision scenarios on Flink – one of the most advanced platforms for stream processing.
+
+## Why Nussknacker
+
+Nussknacker promises to make developing and deploying real-time decision algorithms as easy as it is with spreadsheets and data at rest. If so, domain experts and developers can focus on tasks that each of these two groups is most happy to perform - domain experts can author the decision algorithms and developers can solve problems beyond the reach of tools like Nussknacker.
+
+We discovered that several factors heavily influence the development of algorithms that work with real-time data:
+- Often, these are domain experts who conceptualize the algorithms and the expertise required is very domain-specific. Without proper tools for converting algorithms to code, domain experts have to delegate this work to programmers who are proficient in multiple tools, programming languages, and technologies. This approach costs money and takes time. With Nussknacker domain experts can build the algorithm from prefabricated blocks. The trick is to make these prefabricated blocks infinitely flexible -  Nussknacker achieves this by using [SpEL](https://nussknacker.io/documentation/docs/scenarios_authoring/Intro/#spel), an easy-to-learn expression language, to express data transformations and control the flow of the algorithm.
```

Comment:
Applied suggestions 1-4.

Regarding _infinitely_ vs _really_ - low code tools tend to limit strongly what you can achieve with them. For this reason, to convey the message, even if it may sound like an outrageous claim, I am ready to use the word _infinitely_